README.md

```diff
@@ -303,26 +318,41 @@ Access Sonatype's proprietary vulnerability data using `jake`:
 ```
 > jake iq --help
 
-usage: jake iq [-h] -s https://localhost:8070 -i APP_ID -u USER_ID -p PASSWORD [-t STAGE]
+usage: jake iq [-h] -s https://localhost:8070 -i APP_ID -u USER_ID -p PASSWORD [-st STAGE]
```

Comment:
So is it ok to also change `sbom`'s `-i` option to `-f`? I thought that I had to always use the `-i` flag for uniformity (and to DRY the code), but I just started using jake and don't really know what are the most used flags that are worth keeping.

You're right I should have used the `--help` flag to update the docs, I was boing too lazy!

Reply:
So I went for the `-f` route for every command, and reverted what I could to before. `-st` has to stay to avoid conflicts with `--type` shorthand.
`iq` fails if the `-t` flag is used for stage since it only accepts the supported file formats.
Now the docs are in sync with the code!